usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -52,6 +52,8 @@ This endpoint returns a list of data that is associated with the award profile p
     Award id
 + `parent_award_piid`: `1301` (required, string, nullable)
     null if the contract has no parent
++ `parent_generated_unique_award_id`: `CONT_IDV_W56HZV10AA913_9700` (required, string, nullable)
```

Comment:
Since the contract says it is `nullable` and `required` does it actually return null for assistance awards?

Reply:
It's nullable for the contract response, it's not required for the assistance response. It currently does not return at all for the assistance response, as the assistance awards don't have parent awards.

Reply:
- `required` means the key needs to be present in the API response for the contract to pass.
- nullable allows null to be returned.

Reply:
@emily-brents Assistance awards don't have parents, that is applicable only to procurements and it is for IDVs

Reply:
is that not what i said? it's required but nullable for the contract response, since some contracts don't have parents, and it's not required at all for assistance because they don't have parents.

Reply:
yes....